x-pack/plugins/security_solution/public/common/components/event_details/cti_details/enrichment_accordion_group.tsx

```diff
@@ -65,7 +70,21 @@ const ThreatDetailsDescription: React.FC<ThreatDetailsRow['description']> = ({
   );
 };
 
-const columns: Array<EuiBasicTableColumn<SummaryRow>> = getSummaryColumns(ThreatDetailsDescription);
+const columns: Array<EuiBasicTableColumn<ThreatDetailsRow>> = [
```

Comment:
Do we need the test to check if the right columns is displayed?

Reply:
Yea. I think @janmonschke made these changes to separate the two tables (enrichment and the alert summary one), but we can add tests, I'm just not sure what the shape of the test data should be off hand. Maybe @ecezalp can help out here with test data, and I can add a simple test for this file? I can also do it in a follow up PR if that's easier

Reply:
Jep, I made sure those views still rendered the same